### ISSUE TITLE
Primary colour on charts does not meet accessibility requirements

### ISSUE THREAD

**Issue by bazzadp** [HTTPArchive/almanac.httparchive.org#1756]
![Typical Web Almanac chart from CSS chapter](https://user-images.githubusercontent.com/10931297/102008572-8b99d480-3d29-11eb-8849-9442cf0a922e.jpeg)

The greeny-coloured chart does not meet WCAG AAA (not even AA) colour contrast rules for text meaning when we use a label (like in this example) we fail these criteria.

Particularly annoying when trying to copy edit a section on colour contrast in the Accessibility chapter! I eventually cheated and used inside labels and set it to black:

![Colour Contrast Chart from Accessibility Chapter](https://user-images.githubusercontent.com/10931297/102008636-f519e300-3d29-11eb-8cb5-0d4dca98a22a.jpeg)

How should we handle this? Is it acceptable because the darker label is there (on many charts we drop the lighter label for space reasons). Also the data is typically given in the description. Should we make a note in the Accessibility Statement?

For next year should we change our colour palette or provide guidance on using this label?

**Comment by rviscomi** [comment]
Does that take font size/weight into account? We have some "off black" theme colors so if needed I'd prefer we reuse those instead.

I agree that we're due for a fresh color palette in 2021 anyway, so we could resolve it then.

**Comment by bazzadp** [comment]
Yes it does take that into account. The greeny colour is #a8caba so when compared to background colour of white that's a 1.77 contrast ratio.

WCAG AA demands 3:1 for large text, 4.5:1 for everything else.
WCAG AAA (which we try to follow) demands 4.5:1 for large text and 7:1 for everything else.

So either way we are falling far short.

Agree, it's a bit late to change this now, and it really is supplementry info given we often drop this axis label and have it in our descriptions but still, let's revisit for next year.

Will close for now.
